docs/content/doc/upgrade/from-gitea.en-us.md

```diff
@@ -0,0 +1,86 @@
+---
+date: "2021-09-02T16:00:00+08:00"
+title: "Upgrade from an old Gitea"
+slug: "upgrade-from-gitea"
+weight: 10
+toc: false
+draft: false
+menu:
+  sidebar:
+    parent: "upgrade"
+    name: "From Gitea"
+    weight: 10
+    identifier: "upgrade-from-gitea"
+---
+
+# Upgrade from an old Gitea
+
+**Table of Contents**
+
+{{< toc >}}
+
+To update Gitea, download a newer version, stop the old one, perform a backup, and run the new one.
+Everytime a Gitea instance runs, it checks whether an upgrade action should be taken. 
+If an upgrade action is required, Gitea will take some time to complete the upgrade and then serve.
+
+## Backup for downgrade
+
+Gitea keeps compatibility for patch versions whose first two fields are the same (`a.b.x` -> `a.b.y`), 
+these patch versions can be upgraded and downgraded with the same database structure. 
+Otherwise (`a.b.?` -> `a.c.?`), a newer Gitea version will upgrade the old database 
+to a new structure that may differ from the old version.
+
+For example:
+
+| From | To | Result |
+| --- | --- | --- |
+| 1.4.0 | 1.4.1 | ✅ |
```

Comment:
I'm not sure if we should include information about downgrades (even between patch versions) as we highly recommend against them. The information in this section about backups is very useful though.

Reply:
Changed: ⚠️ Not recommended, take your own risk! Although it may work if the database structure doesn't change, it's highly recommended to use a backup to downgrade.